### PR TITLE
Expose timm constructor arguments

### LINKED ISSUE #954: Allow kwargs in TimmUniversalEncoder
In my opinion, the most attractive use case for certain timm encoders such as ResNet-18, which is also available in torchvision, is that timm generally allows for various additional configuration parameters to be passed to the constructor, such as anti-aliasing, attention, and stochastic depth.

However, smp does not support this this feature at the moment. This is because TimmUniveralEncoder has a local list of kwargs that it passed off to timm and does not accept any others in its initializer.

A very easy fix for this would be to allow the initializer to accept its own kwargs and join them to the corresponding local variable before making any calls to timm.

### PR DESCRIPTION
**Pull request by DimitrisMantas**
Resolves #954

### CODE REVIEW

**Comment by DimitrisMantas** [comment]
I kind of need this, and the community appears to do so as well. What is missing to make it land?

File: segmentation_models_pytorch/encoders/timm_universal.py [hunk 1]
```diff
@@ -36,3 +48,11 @@ def out_channels(self):
...
     def output_stride(self):
         return min(self._output_stride, 2**self._depth)
+
+
+def _merge_kwargs_no_dupls(a: dict[str, Any], b: dict[str, Any]) -> dict[str, Any]:
+    dupls = a.keys() & b.keys()
+    if dupls:
+        raise ValueError(f"'{dupls}' already specified internally")
+
+    return a | b
```
Comment (qubvel) [inline segmentation_models_pytorch/encoders/timm_universal.py:58]: Let's refactor it a bit:
1) dupls -> duplicates
2) "|" not sure which python version introduced it, maybe `{**a, **b}` would be more robust

**Review by qubvel — COMMENTED** [review]
Thanks for the contribution! Overall looks good to me, just a nit comment

File: segmentation_models_pytorch/encoders/timm_universal.py (hunk 1 above)
Comment (adamjstewart) [reply segmentation_models_pytorch/encoders/timm_universal.py:58]: 2. Looks like it was added in Python 3.9, so no need to change this: https://peps.python.org/pep-0584/

**Review by qubvel — APPROVED** [review]
Thanks for the update!

**Pull request merged**